media/client/ipc/include/MediaPipelineIpc.h

```diff
@@ -107,7 +107,7 @@ class MediaPipelineIpc : public IMediaPipelineIpc, public IpcModule
 
     bool flush(int32_t sourceId, bool resetTime) override;
 
-    bool setSourcePosition(int32_t sourceId, int64_t position) override;
+    bool setSourcePosition(int32_t sourceId, int64_t position, bool resetTime) override;
```

Comment:
Again, should position perhaps be uint64_t because the call to gstSegmentDoSeek is unsigned?